### PR TITLE
fix: added meta tag viewport for GH Pages

### PR DESCRIPTION
**Pull request by abarmina**
After adding meta tag 'viewport' a layout change was required, so changes were made to the styles

### CODE REVIEW

File: pages/views/basic/esl-media.njk [hunk 1]
```diff
@@ -12,19 +12,19 @@ tags: basic
 <hr/>
 <div class="row justify-content-around">
   <section class="col-12 row" data-demo="0">
-    <div class="demo col-6">
+    <div class="col-md-6 order-md-2">
+      <h3 class="text-center">ESLMedia Playgroud </h3>
+      <test-media-source target="[data-demo='0'] esl-media"></test-media-source>
+    </div>
+        <div class="demo col-md-6 order-md-1">
```
Comment (ala-n) [inline pages/views/basic/esl-media.njk:19]: Super cosmetic: alignment (extra spaces).

**Review by ala-n — APPROVED** [review]
Superb

**Review by dshovchko — APPROVED** [review]

**Review by YanaBr — APPROVED** [review]

**Review by Sisha3342 — APPROVED** [review]

**Review by nattallius — APPROVED** [review]

File: pages/views/basic/esl-media.njk (hunk 1 above)
Comment (abarmina) [reply pages/views/basic/esl-media.njk:19]: thanks!

**Review by julia-murashko — APPROVED** [review]

**Review by andreybelous — APPROVED** [review]

**Pull request merged**